src/store/main/sagas/loadData.ts

```diff
@@ -37,6 +38,8 @@ export function* loadData() {
 			call(checkIfApproveIbAlluoBTC),
 			call(sweepQueryFlow),
 		]);
+		const endTime = performance.now();
+		console.log('total time to load app', endTime - startTime);
```

Comment:
@Seroxdesign you can take this out, Use Google Chrome > Inspector > Network and you can get these measurements better, I think